openpype/hosts/maya/plugins/publish/collect_yeti_rig.py

```diff
@@ -128,7 +128,14 @@ def get_yeti_resources(self, node):
             image_search_paths = self._replace_tokens(image_search_paths)
 
             # List all related textures
-            texture_filenames = cmds.pgYetiCommand(node, listTextures=True)
+            texture_nodes = cmds.pgYetiGraph(
+                node, listNodes=True, type="texture")
+            texture_filenames = [
+                cmds.pgYetiGraph(
+                    node, node=texture_node,
+                    param="file_name", getParamValue=True)
+                for texture_node in texture_nodes
+            ]
```

Comment:
Isn't this exactly what `listTextures=True` argument for `pgYetiCommand` was supposed to do, but also supporting other usages of textures that are not only in `texture` nodes?

Reply:
> Isn't this exactly what `listTextures=True` argument for `pgYetiCommand` was supposed to do, but also supporting other usages of textures that are not only in `texture` nodes?

for some reasons, the old pgYetiCommand doesn't get any textures list out in version `4.1.4`

Reply:
If it's removed/deprecated I suppose swapping it over should be fine. Thanks.